{"uuid":"4c3b549b-2fd5-49ae-8956-f9c8a8027437","historyId":"GetMakeNegative","fullName":"GetMakeNegative","labels":[{"name":"package","value":"autochek_api_negative.DynamicResponse.DynamicResponseNegative"},{"name":"testClass","value":"autochek_api_negative.DynamicResponse.DynamicResponseNegative"},{"name":"testMethod","value":"GetMakeNegative"},{"name":"parentSuite","value":"API_Regression_Suite"},{"name":"suite","value":"Autochek_API_Automation"},{"name":"subSuite","value":"autochek_api_negative.DynamicResponse.DynamicResponseNegative"},{"name":"host","value":"LAPTOP-NV5LHH0B"},{"name":"thread","value":"19536@LAPTOP-NV5LHH0B.main(3)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"},{"name":"story","value":"Get Make"},{"name":"severity","value":"critical"}],"links":[],"name":"GetMakeNegative","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\tStatus code should match | Expected: 400, Actual: 200 expected [400] but found [200]","trace":"java.lang.AssertionError: The following asserts failed:\n\tStatus code should match | Expected: 400, Actual: 200 expected [400] but found [200]\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)\r\n\tat Listener.AllureLogs.executeSoftAssertAll(AllureLogs.java:201)\r\n\tat autochek_api_negative.DynamicResponse.DynamicResponseNegative.GetMakeNegative(DynamicResponseNegative.java:119)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:565)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1604)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","description":"Validates GET /v1/inventory/make with dynamic fields filtering - Positive Test","steps":[{"name":"SoftAssert: Verify {fieldName} - Expected: '400', Actual: '200'","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[{"name":"softAssert","value":"org.testng.asserts.SoftAssert@361c008"},{"name":"actual","value":"200"},{"name":"expected","value":"400"},{"name":"message","value":"Status code should match"}],"start":1758018295418,"stop":1758018295418},{"name":"Execute all soft assertions","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"The following asserts failed:\n\tStatus code should match | Expected: 400, Actual: 200 expected [400] but found [200]","trace":"java.lang.AssertionError: The following asserts failed:\n\tStatus code should match | Expected: 400, Actual: 200 expected [400] but found [200]\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)\r\n\tat org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)\r\n\tat Listener.AllureLogs.executeSoftAssertAll(AllureLogs.java:201)\r\n\tat autochek_api_negative.DynamicResponse.DynamicResponseNegative.GetMakeNegative(DynamicResponseNegative.java:119)\r\n\tat java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:565)\r\n\tat org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)\r\n\tat org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)\r\n\tat org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)\r\n\tat org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)\r\n\tat org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)\r\n\tat org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)\r\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1604)\r\n\tat org.testng.TestRunner.privateRun(TestRunner.java:741)\r\n\tat org.testng.TestRunner.run(TestRunner.java:616)\r\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:421)\r\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)\r\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)\r\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:312)\r\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\r\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)\r\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)\r\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1208)\r\n\tat org.testng.TestNG.runSuites(TestNG.java:1112)\r\n\tat org.testng.TestNG.run(TestNG.java:1079)\r\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:65)\r\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:105)\r\n"},"stage":"finished","steps":[],"attachments":[{"name":"Soft Assertion Failures","source":"b594f43f-1dd4-4fdf-956c-bd68901d0708-attachment.txt","type":"text/plain"}],"parameters":[{"name":"softAssert","value":"org.testng.asserts.SoftAssert@361c008"}],"start":1758018295418,"stop":1758018295420}],"attachments":[{"name":"Request Log","source":"ae7058c6-a299-4a25-82be-af09f3a64a6c-attachment","type":"application/json"},{"name":"Response Log","source":"950f6e36-e340-4d37-9bd7-62ac047efe68-attachment","type":"application/json"}],"parameters":[],"start":1758018294889,"stop":1758018295428}